src/resources/i18n/translations/en.json

```diff
@@ -260,20 +260,20 @@
   "Custom API domain": "Custom API domain",
   "Custom Feeds": "Custom Feeds",
   "Custom RSS feeds to parse in parallel": "Custom RSS feeds to parse in parallel",
-  "Custom RSS feeds to parse in parallel helper text": "Parsing too many feeds in parallel may cause issues on some devices. 3 is recommended.",
+  "Custom RSS feeds to parse in parallel helper text": "Parsing more than three feeds at once may cause issues on some devices.",
   "Custom web domain": "Web domain",
-  "Custom web domain subtext": "Custom domains to use an alternate server for your app data.",
+  "Custom web domain subtext": "These allow choice of app data server.",
   "Dark Mode": "Dark",
   "Data Settings": "Data Settings",
-  "Default downloaded episode limit for each podcast": "Max downloaded episodes for each podcast",
+  "Default downloaded episode limit for each podcast": "Limit",
```

Comment:
Not sure about this (in Settings → Downloads).
It isn't clear whether this is a setting for the above toggle, but if so it should be grayed out if not selected…?

Reply:
How about "Downloaded episode limit"?

If we just say "Limit" people could think it is "limiting" something else, like the number of concurrent episode downloads allowed.

Reply:
As for graying out when not selected...I would move that to a Github issue with a "ux" label.

Reply:
https://github.com/podverse/podverse-web/issues/1027